arcgis-ios-sdk-samples/Layers/Browse WFS layers/README.md

```diff
@@ -0,0 +1,42 @@
+# Browse WFS layers
+
+This sample demonstrates how to browse a WFS service for layers and add them to the map.
+
+Services often have multiple layers available for display. For example, a feature service for a city might have layers representing roads, land masses, building footprints, parks, and facilities. A user can choose to only show the road network and parks for a park accessibility analysis.
+
+![](image1.png)
+
+## How to use the sample
+
+Select a layer from the list to display it on the map. 
+
+Some WFS services return coordinates in X,Y order, while others return coordinates in lat/long (Y,X) order. If you don't see features rendered or you see features in the wrong location, use the swap switch to change the coordinate order and reload.
+
+## How it works
+
+* Create an instance of `AGSWFSService` with a URL to a WFS feature service. 
```

Comment:
I'm sorry for not being clear. When I said "You might consider changing the sublist to an unordered list", what I meant was the X.1 list items ("Set the axis order…", "The sample uses…"). Please keep this list as an ordered list to match all the other samples.